src/main/java/org/mdbenefits/app/submission/conditions/HasMedicalExpenses.java

```diff
@@ -8,6 +8,6 @@ public class HasMedicalExpenses extends BasicArrayCondition {
 
     @Override
     public Boolean run(Submission submission) {
-        return checkHasValidSelection(submission, "householdMedicalExpenses[]");
+        return checkHasValidSelection(submission, "medicalExpenses[]");
```

Comment:
this was the missing piece.

Reply:
nice catch! I kept running into issues around the name change.